packaging/nuget/nservicebus.ravendb.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe ravendb</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[5.2.8, 6.0.0)" />
-      <dependency id="RavenDB.Client" version="[3.0.3800, 4.0.0)" />
+      <dependency id="RavenDB.Client" version="[3.0.3800, 3.1.0)" />
```

Comment:
We've verified that it doesn't and can't work with Raven 3.5 (they moved methods from an inherited class to a base class) and that there's no way they can fix it that will make it work statically compiled with both versions. So it's really an acknowledgement of reality.